js/api/eclipsefdn.adopters.js

```diff
@@ -168,7 +191,10 @@ class EclipseFdnAdopters {
           if (typeof adopter['logo'] !== 'undefined') {
             logo = adopter['logo'];
           }
-          if (opts['logo_white'] === true && typeof adopter['logo_white'] !== 'undefined') {
+          if (
+            opts['logo_white'] === true &&
```

Comment:
Same as above comment, we shouldn't break Ifs up like this, just makes it take more space and sometimes be harder to read